hpOneView/resources/networking/logical_interconnects.py

```diff
@@ -233,3 +235,14 @@ def get_telemetry_configuration(self, telemetry_configuration_uri):
 
         """
         return self._client.get(telemetry_configuration_uri)
+
+    def get_firmware(self, id_or_uri):
+        """
+        Gets the installed firmware for a logical interconnect.
+
+        Returns:
+            dict:
```

Comment:
You forgot to describe the arguments. Also would be nice give more details about the 'return'